src/Mod/Fem/Gui/Resources/ui/MeshGmsh.ui

```diff
@@ -53,6 +53,9 @@
           <property name="enabled">
            <bool>true</bool>
           </property>
+          <property name="toolTip">
+           <string>Set to 0.0 to set size automatically</string>
```

Comment:
Maybe "Use 0.0 to set the size automatically" to avoid the repetition?

Reply:
Updated